packages/components/psammead-headings/README.md

```diff
@@ -39,7 +39,7 @@ const Wrapper = () => (
 );
 ```
 
-`SubHeading` components can be used as page anchors when passed an `id` prop. To take the above usage as an example:
+`SubHeading` and `PageHeading` components can be used as page anchors when passed an `id` attribute. To take the above usage as an example:
```

Comment:
Should we also add the following? "Note, to make them focusable by assitive technology they will also need a tab index attribute with a value of -1."

Reply:
It is already mentioned in further down https://github.com/bbc/psammead/pull/3489/files#diff-d3f2566006f1e21e5760acc50bc7db4dR62